diagnostic.md

```diff
@@ -157,7 +191,16 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+div will be floated on the left side of the page
+divOne will be floated on the right side of the page
+divTwo will be under div with a gap but not floated
+DivThree will be right next to divTwo on the right side
+divFour will be on the left side but not floated
+divFive will be right under divFour connected and touching it
```

Comment:
Correct order is:
2,3,1
4,5

Since divs 2,3,1 are all floated, they will stay on the same line. Div4 is cleared, but then floated again due to `div` having float and div 5 would appear next to div 4 since it's floated too.